### PR TITLE
adds cleanup job for operator resources

### PR DESCRIPTION
**Pull request by f41gh7**
https://github.com/VictoriaMetrics/helm-charts/issues/131

### CODE REVIEW

File: charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{{- if and .Values.operator.cleanupCRD }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "victoria-metrics-k8s-stack.cleanupHookName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "victoria-metrics-k8s-stack.labels" . | nindent 4 }}
```
Comment (k1rk) [inline charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml:8]: either remove newline or replace `nindent` with `indent`

File: charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+      {{ include "victoria-metrics-k8s-stack.labels" . | nindent 8 }}
```
Comment (k1rk) [inline charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml:18]: same here

Comment (k1rk) [reply charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml:18]: actually applies to all labels below

**Review by k1rk — APPROVED** [review]
minor note on labels. but else LGTM

Thanks for this @f41gh7

File: charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml (hunk 1 above)
Comment (f41gh7) [reply charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml]: Thanks!

File: charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml (hunk 2 above)
Comment (f41gh7) [reply charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/uninstall_hook.yaml]: fixed!

**Pull request merged**